pkg/ccl/changefeedccl/changefeedbase/options.go

```diff
@@ -529,12 +532,17 @@ func (s StatementOptions) IsSet(key string) bool {
 }
 
 // DeprecationWarnings checks for options in forms we still support and serialize,
-// but should be replaced with a new form. Currently hardcoded to just check format.
+// but should be replaced with a new form.
 func (s StatementOptions) DeprecationWarnings() []string {
 	if newFormat, ok := NoLongerExperimental[s.m[OptFormat]]; ok {
 		return []string{fmt.Sprintf(`%[1]s is no longer experimental, use %[2]s=%[1]s`,
 			newFormat, OptFormat)}
 	}
+	for retiredOpt := range RetiredOptions {
+		if _, isSet := s.m[retiredOpt]; isSet {
+			return []string{fmt.Sprintf("%s option is no longer needed", retiredOpt)}
```

Comment:
It'd be nice if we explained what the current behaviour is now that the option is not needed, in this case it always being enabled

Reply:
I think this will have to be done on the docs side; based on the PR description.
(otherwise, we'll have to add more configurations for each option we decide to retire.